java/src/jmri/jmrit/operations/setup/BuildReportOptionPanel.java

```diff
@@ -135,6 +135,10 @@ public void buttonActionPerformed(ActionEvent ae) {
     @Override
     protected void checkBoxActionPerformed(ActionEvent ae) {
         buildReportIndentCheckBox.setEnabled(buildReportCheckBox.isSelected());
+        // use the smallest font to create the longest lines in the build report
+        if (buildReportCheckBox.isSelected()) {
+            fontSizeComboBox.setSelectedItem(7);
```

Comment:
Is it always certain that there are 7 font sizes?  Or could this be `setSelectedItem(fontSizeComboBox.getModel().getSize()-1)` ?

Reply:
Font sizes available to users are 7 through 18. If for some reason we decided that 7 was too small and eliminated that option, then the new code would become a NOP.  The code that creates the font size selection is:

   public static void loadFontSizeComboBox(JComboBox<Integer> box) {
        // load font sizes 7 through 18
        for (int i = 7; i < 19; i++) {
            box.addItem(i);
        }
        Dimension size = box.getPreferredSize();
        size = new Dimension(size.width + 10, size.height);
        box.setPreferredSize(size);
    }